src/lib/mdns/Advertiser.h

```diff
@@ -75,10 +113,13 @@ class ServiceAdvertiser
 
     /// Starts the advertiser. Items 'Advertised' will become visible.
     /// May be called before OR after Advertise() calls.
-    virtual CHIP_ERROR Start(chip::Inet::InetLayer * inetLayer, uint16_t port) = 0;
+    virtual CHIP_ERROR Start(chip::Inet::InetLayer * inetLayer, uint64_t macAddress, uint16_t port, bool enableIPv4) = 0;
```

Comment:
should we call this `macAddress` or is that an implementation detail and it should be some 'unique identifier' instead?